### PR TITLE
Add more debugging

### PR DESCRIPTION
**Pull request by trask**
Also discovered there was an extra space at the begging of each ndjson line, fixed by using `jg.setRootValueSeparator(new SerializedString("\n"))`

attaching SNAPSHOT for testing: [applicationinsights-agent-3.2.4-BETA-SNAPSHOT.jar.zip](https://github.com/microsoft/ApplicationInsights-Java/files/7550217/applicationinsights-agent-3.2.4-BETA-SNAPSHOT.jar.zip)

### CODE REVIEW

File: agent/agent-tooling/src/test/java/com/microsoft/applicationinsights/agent/internal/localstorage/IntegrationTests.java [hunk 1]
```diff
@@ -114,10 +113,10 @@ public void integrationTest() throws Exception {
     }
 
     ExecutorService executorService = Executors.newFixedThreadPool(10);
-    for (int i = 0; i < 10; i++) {
+    for (int i = 0; i < 1; i++) {
```
Comment (heyams) [inline agent/agent-tooling/src/test/java/com/microsoft/applicationinsights/agent/internal/localstorage/IntegrationTests.java:116]: for loop is not needed for 1 iteration.

File: agent/agent-tooling/src/test/java/com/microsoft/applicationinsights/agent/internal/localstorage/IntegrationTests.java [hunk 2]
```diff
@@ -114,10 +113,10 @@ public void integrationTest() throws Exception {
...
       executorService.execute(
           () -> {
-            for (int j = 0; j < 10; j++) {
+            for (int j = 0; j < 1; j++) {
```
Comment (heyams) [inline agent/agent-tooling/src/test/java/com/microsoft/applicationinsights/agent/internal/localstorage/IntegrationTests.java:119]: same here

File: agent/agent-tooling/src/test/java/com/microsoft/applicationinsights/agent/internal/localstorage/IntegrationTests.java [hunk 3]
```diff
@@ -127,9 +126,9 @@ public void integrationTest() throws Exception {
 
     executorService.shutdown();
     executorService.awaitTermination(10, TimeUnit.MINUTES);
-    assertThat(localFileCache.getPersistedFilesCache().size()).isEqualTo(100);
+    assertThat(localFileCache.getPersistedFilesCache().size()).isEqualTo(1);
 
-    for (int i = 100; i > 0; i--) {
+    for (int i = 1; i > 0; i--) {
```
Comment (heyams) [inline agent/agent-tooling/src/test/java/com/microsoft/applicationinsights/agent/internal/localstorage/IntegrationTests.java:131]: same here

File: agent/agent-tooling/src/test/java/com/microsoft/applicationinsights/agent/internal/localstorage/IntegrationTests.java (hunk 1 above)
Comment (trask) [reply agent/agent-tooling/src/test/java/com/microsoft/applicationinsights/agent/internal/localstorage/IntegrationTests.java]: oops! this was just a local change for testing, reverted, thx!

**Review by heyams — APPROVED** [review]

**Review by kryalama — APPROVED** [review]

**Pull request merged**